library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
@gebner I'm missing something basic about `antiquote_pexpr` approach. We have definitions such as:
```lean
meta def repeat : itactic → tactic unit :=
tactic.repeat
```
It type checks because `itactic` is definitionally equal to `tactic`. The definition of `itactic` you gave above isn't.